Tasks/VsTestV2/inputdatacontract.ts

```diff
@@ -25,6 +25,7 @@ export interface TestReportingSettings {
     TestResultsDirectory : string;
     TestRunSystem : string;
     TestSourceSettings : TestSourceSettings;
+    ExecutionStatusSettings: MinimumTestsExecutionStatusSettings;
```

Comment:
this goes under execution settings. same change in the c# side

Reply:
This has been added in TestReportingSettings only in c# as well.